lib/src/values/small_logic_value.dart

```diff
@@ -180,4 +180,18 @@ class _SmallLogicValue extends LogicValue {
               _mask,
           _invalid >> shamt,
           width);
+
+  @override
+  BigInt get _bigIntInvalid =>
+      BigInt.from(_invalid) & _BigLogicValue._maskOfWidth(width);
+
+  @override
+  BigInt get _bigIntValue =>
+      BigInt.from(_value) & _BigLogicValue._maskOfWidth(width);
```

Comment:
Why is this masking needed?  No masking is needed when returning a plain `_intValue`, so `BigInt.from(_value)` should be already correct, no?

Reply:
I originally didn't have the mask, but it creates a bug when the `_value` can be interpreted as a negative number since when it is bitwise-`|`ed against another bus it sign extends the 1s.